SyncIntegrationTests/Pipfile.lock

```diff
@@ -3,6 +3,19 @@
         "hash": {
             "sha256": "112a12fa2e9e8117b399b60a49b4c8799a614ef655992640c95149bf95f33e8b"
         },
+        "host-environment-markers": {
+            "implementation_name": "cpython",
+            "implementation_version": "0",
+            "os_name": "posix",
+            "platform_machine": "x86_64",
+            "platform_python_implementation": "CPython",
+            "platform_release": "17.7.0",
+            "platform_system": "Darwin",
+            "platform_version": "Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64",
+            "python_full_version": "2.7.14",
+            "python_version": "2.7",
+            "sys_platform": "darwin"
+        },
```

Comment:
This suggests you're using an old version of pipenv. Could you update and run `pipenv update` again? If you installed using Homebrew you can run `brew update && brew upgrade`.

Reply:
@davehunt I run those commands but do not see any difference on Pipfile.lock
I have pipenv 2018.7.1 which I think is the latest

Reply:
hmm.. I wonder why it's not clearing this section. could you try removing this file and recreating it with `pipenv lock`?

Reply:
👍

Reply:
I can still see that line after re-creating the Pipfile.lock but some hashes have changed, @davehunt is it still wrong? 
Thanks!

Reply:
No idea why it is still shown :( but agree, does not seem to affect the tests, they are green in this PR. Thanks! enjoy your PTO!